Podcast/SharedContentViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SharedContentViewController.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/15/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import NVActivityIndicatorView
+import SnapKit
+
+// View Controller showing all content personally shared with you by other users
```

Comment:
nice! use a triple slash (`///`) here so it comes up in documentation

Reply:
done